### PR TITLE
style(pthi): amtinfo command add amt ipaddr and osipaddr

### PR DESCRIPTION
**Pull request by tongsean9807**

### CODE REVIEW

**Comment by tongsean9807** [comment]
Hi,
I will set it to return 0. amt by default return 0 is the struct is empty

I have pass test a well.
image

**Comment by bgmckown** [comment]
Am I overlooking something obvious here? This appears to be a redundant PR with https://github.com/tongsean9807/rpc-go/pull/1

File: internal/amt/commands.go [hunk 1]
```diff
@@ -42,6 +42,7 @@ type InterfaceSettings struct {
 	DHCPEnabled bool   `json:"dhcpEnabled"`
 	DHCPMode    string `json:"dhcpMode"`
 	IPAddress   string `json:"ipAddress"` //net.IP
+	OsIPAdress  string `json:"osIpAddress"`
```
Comment (bgmckown) [inline internal/amt/commands.go]: spelling correction
```suggestion
	OsIPAddress  string `json:"osIpAddress"`
```

File: internal/amt/commands.go [hunk 2]
```diff
@@ -351,6 +352,7 @@ func (amt AMTCommand) GetLANInterfaceSettings(useWireless bool) (InterfaceSettin
 
 	settings := InterfaceSettings{
 		IPAddress:   "0.0.0.0",
+		OsIPAdress:  "0.0.0.0",
```
Comment (bgmckown) [inline internal/amt/commands.go]:
```suggestion
		OsIPAddress:  "0.0.0.0",
```

File: internal/local/info.go [hunk 3]
```diff
@@ -165,7 +165,8 @@ func (service *ProvisioningService) DisplayAMTInfo() (err error) {
 			service.PrintOutput("DHCP Enabled 		: " + strconv.FormatBool(wired.DHCPEnabled))
 			service.PrintOutput("DHCP Mode    		: " + wired.DHCPMode)
 			service.PrintOutput("Link Status  		: " + wired.LinkStatus)
-			service.PrintOutput("IP Address   		: " + wired.IPAddress)
+			service.PrintOutput("AMT IP Address   	: " + wired.IPAddress)
+			service.PrintOutput("OS  IP Address		: " + wired.OsIPAdress)
```
Comment (bgmckown) [inline internal/local/info.go]:
```suggestion
			service.PrintOutput("OS  IP Address		: " + wired.OsIPAddress)
```

File: internal/local/info.go [hunk 4]
```diff
@@ -179,7 +180,8 @@ func (service *ProvisioningService) DisplayAMTInfo() (err error) {
 		service.PrintOutput("DHCP Enabled 		: " + strconv.FormatBool(wireless.DHCPEnabled))
 		service.PrintOutput("DHCP Mode    		: " + wireless.DHCPMode)
 		service.PrintOutput("Link Status  		: " + wireless.LinkStatus)
-		service.PrintOutput("IP Address   		: " + wireless.IPAddress)
+		service.PrintOutput("AMT IP Address   	: " + wireless.IPAddress)
+		service.PrintOutput("OS  IP Address		: " + wireless.OsIPAdress)
```
Comment (bgmckown) [inline internal/local/info.go]:
```suggestion
		service.PrintOutput("OS  IP Address		: " + wireless.OsIPAddress)
```

**Review by bgmckown — CHANGES_REQUESTED** [review]

**Review by bgmckown — APPROVED** [review]

**Pull request merged**